src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -912,6 +918,60 @@ void tlrRequestCreatedWhenTlrFeatureEnabled() {
       getUserContextMatchers(requester)));
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"HOLD,PAGE", "PAGE"})
+  void canCreateHoldTitleLevelRequestWithRequestPolicyNotAllowingRecallsOrRecallsAndHolds(String allowedRequestTypes) {
```

Comment:
canCreateHoldTLRWhenNotAllowedByRequestPolicy

Reply:
Name is changed accordingly.